### PR TITLE
Fixing PB entrypoint

### PR DESCRIPTION
**Pull request by novicecpp**
Fix https://github.com/dmwm/CRABServer/issues/8561

### CODE REVIEW

File: src/python/Publisher/TaskPublishRucio.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 from TaskWorker.WorkerExceptions import CannotMigrateException
 from TaskWorker.WorkerUtilities import getCrabserver
 
-from PublisherUtils import setupLogging, prepareDummySummary, saveSummaryJson, \
+from Publisher.PublisherUtils import setupLogging, prepareDummySummary, saveSummaryJson, \
```
Comment (belforte) [inline src/python/Publisher/TaskPublishRucio.py:18]: I never understood those things fully. Why you need to add `Publisher.` prefix here but not in the following line ?

Comment (belforte) [reply src/python/Publisher/TaskPublishRucio.py:18]: comparing with file above (I was reading bottom up apparently), it seems an oversight

**Review by belforte — APPROVED** [review]
Looks very good. Thanks

File: src/python/Publisher/TaskPublishRucio.py (hunk 1 above)
Comment (novicecpp) [reply src/python/Publisher/TaskPublishRucio.py:18]: Nice catch!
Thanks

**Comment by novicecpp** [comment]
Thanks for review Stefano.
Waiting for https://gitlab.cern.ch/crab3/CRABServer/-/pipelines/7820917 to finish.

**Comment by belforte** [comment]
I am looking at pylint errors.

**Comment by belforte** [comment]
1. due to "import confusion" pylint did not spot those issues until now. but issues are real bugs
2. those are only present in Publisher_Rucio and would only be triggered when something goes wrong, which never happens in our tests since we do same thing over and over
3. at this point it is better do it while also pursuing more commonality between pub_schedd and pub_rucio, i.e. changing things a bit
4. so I will make a PR in CRABServer repo to fix the bugs and then you can merge into your branch. Seems simpler than making a PR for your repository

**Pull request merged**